Dockerfile

```diff
@@ -1,6 +1,10 @@
 FROM node:18
 WORKDIR /app
 COPY . .
+WORKDIR /app/client
+RUN npm ci
+RUN npm install -g @angular/cli
+RUN ng build
```

Comment:
nit: personally, I'd recommend adding some blank lines to divide the Dockerfile into sections: I had one section for the initial WORKDIR and copy, one section for the stuff on the client app, one section for the stuff on the server app, and then the final EXPOSE/CMD.